pkg/api/vm.go

```diff
@@ -137,7 +137,7 @@ func (v *VMDef) adjustNetBootIdx(qti *qcli.QemuTypeIndex) ([]string, error) {
 	// for any nics without a BootIndex, allocate one
 	for n := range v.Nics {
 		nic := v.Nics[n]
-		if nic.BootIndex == "" {
+		if nic.BootIndex == "" && nic.BootIndex != "off" {
```

Comment:
This line isn't right.  If nic.BootIndex == "" then it clearly != "off".  Should it just be un-changed?